fulp_modules/features/antagonists/bloodsuckers/code/powers/fortitude.dm

```diff
@@ -82,7 +82,7 @@
 /datum/action/bloodsucker/fortitude/hunter
 	name = "Flow"
 	desc = "Use the arts to Flow, giving shove and stun immunity, as well as brute, burn, dismember and pierce resistance. You cannot run while this is active."
-	purchase_flags = HUNTER_CAN_BUY
+	purchase_flags = NONE
```

Comment:
was this proc ever used? does the monster hunter not use this?

Reply:
Monster hunters have the action granted to them, they never purchase powers so this did nothing.